agent/consul/state/catalog.go

```diff
@@ -3077,7 +3081,7 @@ func (s *Store) ServiceTopology(
 		downstreamSources[svc.Name.String()] = source
 	}
 
-	idx, unfilteredDownstreams, err := s.combinedServiceNodesTxn(tx, ws, downstreamNames)
+	idx, unfilteredDownstreams, _, err := s.combinedServiceNodesTxn(tx, ws, downstreamNames)
```

Comment:
Currently, services infer their downstreams _through_ routing configs (i.e. they are not aware of routing configs and point directly to their downstreams).
Not sure if we want to expose routing config downstreams here as well?